src/sql/src/statements/tql.rs

```diff
@@ -55,6 +97,23 @@ pub struct TqlAnalyze {
     pub is_verbose: bool,
 }
 
+impl Display for TqlAnalyze {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "TQL ANALYZE ")?;
+        if self.is_verbose {
+            write!(f, "VERBOSE ")?;
+        }
+        write!(f, "(")?;
+        write!(f, "{}, ", &self.start)?;
+        write!(f, "{}, ", &self.end)?;
+        write!(f, "{}", &self.step)?;
+        if let Some(lookback) = &self.lookback {
+            write!(f, ", {}", lookback)?;
+        }
```

Comment:
Looks like this code block appears three times in this file, could we extract it with a macro?